modules/firstMessage.js

```diff
@@ -0,0 +1,27 @@
+const addReactions = (message, reactions) => {
```

Comment:
What does this file do? I can't really figure it out

Reply:
it adds reactions to the first message of a channel.  i added you on discord.  acept when u can and I'll invite u to my dev server and I can show u how the bot works

Reply:
can you merge the firstMessage.js with roleReaction.js? Is there a specific reason they need to be two different files?